### ISSUE TITLE
Deaths in car accidents in Greece[Project]

### ISSUE THREAD

**Issue by chrgavalas** [jsoma/data-studio-projects#15]
Please complete all of the following sections, or a robot will spookily dance around your issue! A completed version of this template can be found at https://github.com/jsoma/data-studio-projects/issues/1

# Pitch
In the last few months there's been a huge discussion in Greece about safety in highways due to a horrific car accident caught on video that caused four people's death. But is this really the case? 

## Summary

A shocking accident with a Porsche out of control on the highway Athens - Lamia in central Greece in which three young adults and a toddler found tragic death has spurred an extensive discussion in the -stricken by deep recession country- about whether the roads are safe. In 2010 data was showing that the country presented the highest car accident rate in the EU with 110 deaths per one million citizens. Now the figures are down, but is it truly getting better? Through the examination of the real numbers compared with other European countries of the same population, as well as of the investments in infrastructure and of whether Greek keep buying new cars or have turned to used ones, we sort of start making valid assumptions on the problem’s real scope/dimensions. 

## Details

Greece needs serious reinvigoration of its infrastructure and that’s something the EU is currently helping with. It announced a 1.3 billion euros investment effort that will allow major projects that started in the 2007-2013 financial period to be completed in the current 2014-2020 period. These projects co-financed by the EU are meant to support a greek led growth strategy, according to high executives in the EU. 

**Possible headline(s):** 
Deadly greek roads: real reasons and solutions to the fatal trend 

**Data set(s):** 
Two datasets from OECD, one from the European Commission  
https://data.oecd.org/transport/road-accidents.htm
https://ec.europa.eu/transport/road_safety/specialist/statistics_en

**Code repository:** 
https://github.com/chrgavalas/data-projects/tree/master/code/project3
	
**Possible problems/fears/questions:** 
The story is based on valid assumptions based on specific datasets that I could work with and make aggregations. There's datasets I couldn't come across whose existence would have made the story flow much better without some unavoidable leaps. 

## Work so far
-Dataset from OECD with injuries-fatalities of road accidents in various countries, showing most road accident deaths in Greece took place in the late 90s 
-European Commission statistics on exact number of fatalities in European countries since 1991, chance to analyze exact numbers comparatively. Data confirms deaths in road accidents in Greece are on the decline but still higher than what is observed in countries with similar population in Europe.
-Is it maybe because there's a lot of cars in the highways? Data from OECD show that up until 2009 Greeks were buying cars in masses! After the recession hit it was one of the first things to cut back, it looks as if the following years roads were filled with used cars of low maintenance.
-New dataset from OECD with figures about road infrastructure in various countries. Road infrastructure in Greece seems to be relatively low, but slightly higher that the figures about the same set of countries of around the same population. So Greece is investing more than these countries in roads but it still has more deaths in road accidents than them. Why?    
![download-2](https://user-images.githubusercontent.com/30135399/28332959-765cb284-6bc4-11e7-98f9-499afb361690.png)

-Bring in a new indicator to try to explain it: Road fatalities per one million inhabitants which will give us a better comparison with more countries. It becomes obvious that there's an issue here. Death numbers are indeed declining but remain relatively high in Greece! More than in the US! 
-Finally I try to examine if the reason why Greeks stick to their cars despite not buying new ones is the lack of railway infrastructure that would give them a good alternative. New data from OECD shows that investments in railways are declining in Greece after the 2004 Olympics. That seems to not give an incentive to Greeks to abandon their deep rooted preference to a private vehicle, which increases the risk of fatal accidents given the fact that road infrastructure is also, not at the level it should be.
-Maybe that triggered EU interest in investing in major growth-enabling infrastructure in Greece. 

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue issue has `[Project]` in the title
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far
- [x] I have received two comments of peer feedback on my pitch issue

**Comment by RodrigoMenegat** [comment]
- I really like the idea of using data to make the discussion about such an emotional topic more evidence-based. Why are the numbers dropping? Did the government pass some legislation about this issue recently?
- You could compare Greece car crash rates (and their changes over time) with the neighboring countries, as well.

**Comment by amandolinjames** [comment]
This is a very journalistic story- could see it being published in a daily news outlet. Maybe look at injuries and not just deaths.

**Comment by cathrinesot** [comment]
Nice!!! Check also here for some more data!

http://www.astynomia.gr/index.php?option=ozo_content&perform=view&id=70427&Itemid=86&lang=

http://geodata.gov.gr/dataset/trokhaia-atukhemata

**Comment by christiano2301** [comment]
![download-3](https://user-images.githubusercontent.com/28814388/28430965-7ecc603a-6d50-11e7-9d02-e3be42fa8c67.png)

![download-4](https://user-images.githubusercontent.com/28814388/28431019-b2d2b1e0-6d50-11e7-8676-ccdb6dbbfccf.png)

![download-1](https://user-images.githubusercontent.com/28814388/28430395-63a18c74-6d4e-11e7-90d2-19102406286b.png)

![download-5](https://user-images.githubusercontent.com/28814388/28431106-05a18284-6d51-11e7-8509-b80157366e10.png)

![download-4](https://user-images.githubusercontent.com/28814388/28430863-08b718d6-6d50-11e7-827d-9804c2c8e692.png)

![download-5](https://user-images.githubusercontent.com/28814388/28430893-2f11d6a6-6d50-11e7-9f56-b9a51492caea.png)

**Comment by christiano2301** [comment]
# Update
	
## Content
	
![download-3](https://user-images.githubusercontent.com/28814388/28430965-7ecc603a-6d50-11e7-9d02-e3be42fa8c67.png)

![download-4](https://user-images.githubusercontent.com/28814388/28431019-b2d2b1e0-6d50-11e7-8676-ccdb6dbbfccf.png)

![download-1](https://user-images.githubusercontent.com/28814388/28430395-63a18c74-6d4e-11e7-90d2-19102406286b.png)

![download-5](https://user-images.githubusercontent.com/28814388/28431106-05a18284-6d51-11e7-8509-b80157366e10.png)

![download-4](https://user-images.githubusercontent.com/28814388/28430863-08b718d6-6d50-11e7-827d-9804c2c8e692.png)

![download-5](https://user-images.githubusercontent.com/28814388/28430893-2f11d6a6-6d50-11e7-9f56-b9a51492caea.png)

## Any changes in direction or topic?
	
## Problems/Questions
	
## Checklist
	
- [x] I have included my visuals
- [x] I have added writing about my topic
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
- [x] I have received two comments of peer feedback on my project update

**Comment by playfairbot** [comment]
Howdy! This is the Playfair Bot, let's see what's been going on here.

Please post your **project draft!** It should be posted by **Friday**. More details available [here](/jsoma/data-studio-projects). If you posted one but I'm not seeing it, make sure you followed [the template](/jsoma/data-studio-projects/blob/master/UPDATE_TEMPLATE.md).

**Comment by peterjost** [comment]
Nice idea - and what a crisis in Greece after 2008! My suggestion: sort the first graph by year, not by number of deaths. It will be easier to understand the trend.

**Comment by chrgavalas** [comment]
# Final

## Project visuals/text

In the last few months there's been a huge discussion in Greece about safety in highways due to a horrific car accident caught on video that caused four people's death. But is this really the case?

A shocking accident with a Porsche out of control on the highway Athens - Lamia in central Greece in which three young adults and a toddler found tragic death became a leading story for quite some time in national news. The luxurious car lost control before exiting the highway and crashed into a parked car at a rest stop killing a young mother and her 3-year old child. The parked car was hit while in a parking space outside of a public restroom, where the father of the three-year-old and husband of the 33-year-old woman had stopped to use the facilities. They were waiting for him to return when impact occurred.

![copy](https://user-images.githubusercontent.com/30135399/28687963-39c3e538-72de-11e7-89d6-2122114d6eff.jpg)

In early 2016 another car accident came to the spotlight. Pantelis Pantelidis, one of Greece’s most popular young singers, was killed in a car crash. The 32-year-old was driving on a southern Athens avenue when he lost control of his car early in the morning.

![pantelidis](https://user-images.githubusercontent.com/30135399/28688714-ce1077d6-72e0-11e7-9e26-8b3d0b391e20.jpg)

Both incidents have spurred an extensive discussion in the -stricken by deep recession country- about whether the roads are indeed safe. Data from OECD actually indicate that most deadly cars accidents occurred in greek roads in the late 90s. But does this mean things have gotten significantly better?

![pic1](https://user-images.githubusercontent.com/30135399/28688698-bc87c32a-72e0-11e7-89ea-6617639de0ab.png)

Numbers are down since then, but maybe still at a high comparative level?

However, statistics from the European Commission on exact number of fatalities since 1991 give a different perspective. Numbers for the vast majority of European countries are indeed down in comparison to the 90s but deaths from car accidents in Greece are currently higher than any other European nation with similar population. 

![pic2 copy](https://user-images.githubusercontent.com/30135399/28688005-65ab1cb6-72de-11e7-8a4a-9df3a06d9c65.png)

Furthermore, according to PwC’s analysis on infrastructure in the country published in 2017, Greece is ranked 26th among the EU countries in terms of infrastructure quality, along with systematic low infrastructure quality countries, mostly in Southern Europe.

As a matter of fact, when counting road fatalities per one million inhabitants Greece finds itself on the ninth position worldwide, ahead of even the US!

![pic5](https://user-images.githubusercontent.com/30135399/28688752-f3ff951c-72e0-11e7-9e91-32f4877ee9a2.png)

Greeks have long had a love affair with cars

Prior to the economic crisis Greece was among the top 5 countries on earth in the consumption of private vehicles per capita reflecting its citizens' lust for the wheel even if that meant they had to be deprived of other goods. Only Canada, Ireland, Norway and Denmark surpassed Greece up until 2009. In fact, Greeks used to buy more of cars per capita than many of their European neighbors.

But when the economy collapsed, the country became a vast graveyard of abandoned automobiles. In the last few days of each year, many people line up to turn in their license plates in advance of a new tax on cars that the government typically always allows the deadline to stretch into the new year.

![pic33](https://user-images.githubusercontent.com/30135399/28688769-089c689c-72e1-11e7-86c3-621e1f0db561.png)

![pic4 copy](https://user-images.githubusercontent.com/30135399/28688031-7b971160-72de-11e7-8f76-0e01295a89fc.png)

But this development didn't mean Greeks necessarily drove less. It's more likely that during the years that followed, roads got filled mostly with used cars which in many cases haven't been properly repaired as Greeks have been trying to avoid maintenance costs.

According to a February 2017 Kathimerini newspaper report, public organizations keep in use vehicles that should have been scrapped years ago. It is indicative that most ambulances of the National Health System are hardly operational and in many cases they break down on the road carrying a patient. The same applies to Greece’s public transportation fleet, amounting to a total of 2,000 buses. Public transportation officials say that about 1,000 of them are worn out beyond repair due to rust and serious engine problems. The average age of public buses is 13 years, as 40% of the fleet started running between 1998 and 2002. Greek police vehicles are no exception. The total fleet of Hellenic Police consists of about 12,700 vehicles, including motorcycles and about half of them are not operational.

Is Greece maybe underinvesting in road infrastructure?

But not spending money on a new vehicle fleet seems to be going hand by hand with an underinvestment tendency in infrastructure. A new dataset from OECD with figures about road infrastructure in various countries sheds further light on the increasing lack of investments in this field. Europe's low investment rate has apparently left its infrastructure creaking and Greece hasn't been an exception. The country is clearly trailing in the corresponding list in which China, the US, Japan and only Germany from Europe have secured a front seat.

![invest](https://user-images.githubusercontent.com/30135399/28688785-1df9dfee-72e1-11e7-91ef-adcd1c822d9b.png)

Actually, the infrastructure investment gap is between 0.8 percent of GDP (against the European average) or 1.4 percent of GDP (against historical performance) translating into 1.1% or € 2bn new spending per year.

Railway infrastructure: Do Greeks have a real alternative to driving?

Based on data from the OECD we draw the conclusion that investments in railway infrastructure are also declining in Greece after the 2004 Olympics. That seemingly doesn't give an incentive to Greeks to abandon their deep rooted preference to a private vehicle, which increases the risk of fatal accidents given the fact that new investments in roads are also, not at the level they should be.

![pic6](https://user-images.githubusercontent.com/30135399/28688819-3aa23a9c-72e1-11e7-859a-1f4193c57654.png)

In January 2017, Greece wrapped up the sale of its struggling rail operator TrainOSE to Italy’s state railway company as the country’s government came under pressure from bailout lenders to accelerate its privatization program. The sale to Ferrovie dello Stato Italiane will bring in only €45m but estimates are it completes a series of infrastructure sales to international investors that will boost Greece’s role as a transport and tourism hub for the eastern Mediterranean.
EU invests in major growth-enabling infrastructure in Greece

In March 2017 the EU announced that over €1.3 billion of Cohesion Policy funds will be invested in ten broadband, transport and environmental projects in Greece. More specifically, €377 million will be directed to urban public transport systems in Athens and the region of Attica, over €730 million to the extension of the metro in Thessaloniki, while almost €50 million to sustainable mobility in the Peloponnese peninsula, in the south of Greece. Lastly, €92 million will be invested to achieve better connectivity in the North of Greece.

But could there be a positive takeaway from the dire findings? Apparently yes, says the PwC report, according to which infrastructure investments in Greece have an economic multiplier of around 1.8x. That means it can boost demand of other sectors and lead Greek economy to growth. This measurement specifically indicates that for every euro spent on infrastructure, GDP is further increased by €0.8.

<img width="200" alt="new" src="https://user-images.githubusercontent.com/30135399/28688831-4ca1145c-72e1-11e7-9873-3e5dad50690e.png">


## Details

**Headline:** 
Deadly greek roads: the EU jumps to the rescue

**Published website version:**
https://chrgavalas.github.io/data-projects/Deadly-Greek-Roads/

**Code repository:**
https://github.com/chrgavalas/data-projects/tree/master/code/project3

**Final data set(s):** 
https://ec.europa.eu/transport/road_safety/specialist/statistics_en
https://data.oecd.org/transport/road-accidents.htm

## What did you find to be the most difficult part of this project?
The issue has many aspects that should be analyzed properly to give out 100% valid results. For example, I wanted to look into the timeline of investments Greece has been making in the last few years to the detriment of road infrastructure but didn't have time. The possible correlations were numerous and I just ended up working with those which provided a sequence of events that would seemingly make sense to explain the trend. Also, I was not primarily and personally attached to the issue, it just gave me an excellent opportunity to feel comfortable with multiple analyses of dataframes due to the variety of data the OECD had. 

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I'd like to follow up on the investments that were decided for Greece and see whether it improves the situation on the streets or not. My efforts to graph the data took up a significant amount of time I would have otherwise spent to further research my topic and either find new angles or eliminate parts whose dynamic would have been weakened. Also I started working without writing down the challenges I faced on the DIARY.md, which is something I will be including in my second project and onwards. But overall I think it stands as a news piece. 

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
